src/js/vic-v2/config/form.js

```diff
@@ -193,7 +193,22 @@ const formConfig = {
           uiSchema: {
             'ui:description': DD214Description,
             dd214: fileUploadUI('Upload your discharge document', {
-              endpoint: '/v0/vic/vic_attachments'
+              endpoint: '/v0/vic/supporting_documentation_attachments',
+              fileTypes: ['pdf'],
```

Comment:
We should be able to upload other file types.

Reply:
Thanks @jbalboni, changed to include the same types allowed for the ID photo as well as pdf.